docs/02-app/02-api-reference/05-next-config-js/cssChunking.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: cssChunking (Experimental)
```

Comment:
```suggestion
title: cssChunking (experimental)
```

Reply:
Hm, Stales Times is capitalized wrong, and Partial Prerendering should be ppr.

Reply:
Looks like I already fixed this in the RC docs 😅